src/instance/discord/commands/ReconnectCommand.ts

```diff
@@ -0,0 +1,20 @@
+import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js'
+import { DiscordCommandInterface, Permission } from '../common/DiscordCommandInterface'
+import DiscordInstance from '../DiscordInstance'
+
+export default {
+  getCommandBuilder: () => new SlashCommandBuilder().setName('reconnect').setDescription('reconnect minecraft clients'),
+  allowInstance: true,
+  permission: Permission.HELPER,
+
+  handler: async function (clientInstance: DiscordInstance, interaction: ChatInputCommandInteraction) {
+    await interaction.deferReply()
+
+    const targetInstance: string | null = interaction.options.getString('instance')
+    clientInstance.app.emit('restartSignal', {
```

Comment:
change the event name too to match its usage. Changing event will be a breaking change since it is core api related. I'll add a label to indicate that.